test/Robot/YouBotGripping.jl

```diff
@@ -10,8 +11,6 @@ include("$(ModiaLang.path)/models/Electric.jl")
 include("$(ModiaLang.path)/models/Rotational.jl")
 include("$(ModiaLang.path)/models/Translational.jl")
```

Comment:
In some cases it was possilbe to replace `ModiaLang` as well, but here it is needed to include some models.